tests/py/dynamo/conversion/test_expand_aten.py

```diff
@@ -27,6 +28,33 @@ def forward(self, x):
             inputs,
         )
 
```

Comment:
Not a necessary change, but for uniformity maybe we can have. Just would improve readability and keep it uniform across the dynamic and non dynamic test cases.
`def test_expand(self, _, init_size, sizes):`